### PR TITLE
Include `gwcli export` output (bsc#1142791)

### PR DESCRIPTION
**Pull request by tserong**
Signed-off-by: Tim Serong <tserong@suse.com>

### CODE REVIEW

File: ses [hunk 1]
```diff
@@ -288,6 +288,16 @@ if [ -e /var/log/ganesha/ganesha.log ]; then
     plugin_message "NFS-Ganesha logs copied to ceph/log/ganesha/ subdirectory"
 fi
 
+#############################################################
+section_header "iSCSI Gateway config"
+
+if [ -x /usr/bin/gwcli ]; then
+    mkdir -p $LOG/ceph/conf/iscsi
+    plugin_command "gwcli export" |
```
Comment (ricardoasmarques) [inline ses:296]: @tserong Will this command be executed by the `root` user?

Just want to make sure we will not hit this - https://github.com/ceph/ceph-iscsi/blob/master/gwcli.py#L93

Comment (tserong) [reply ses:296]: Yep (supportconfig itself will bail out with "ERROR: You must be logged in as root" if not run as root)

Comment (ricardoasmarques) [reply ses:296]: Thanks

**Review by ricardoasmarques — APPROVED** [review]
lgtm

**Pull request merged**